src/client/components/app/ApyTooltipData/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { useAppTranslation } from '@src/client/hooks';
+import { getConstants } from '@src/config/constants';
+import { formatApy } from '@src/utils';
+import { Apy } from '@types';
+import { FC } from 'react';
+import styled from 'styled-components';
```

Comment:
We have to still write a guideline for this:
- external imports should be at the top (react and styled components) followed by a new line
- prefer directly importing from the direct path if its available. For `@src/client/hooks` do `@hooks`. See `tsconfig.paths.json`

Reply:
1. What's the logic behind that style rule
2. will do!

Reply:
Just a convention to have better organization. Easier to notice what external packages are used instead of going through all.

First import was missed in the fix, as well as new line.